sn_client/src/files.rs

```diff
@@ -83,6 +83,12 @@ impl FilesApi {
         let file_size = metadata.len();
 
         let (head_address, data_map_chunk, mut chunks_paths) =
+
+            // todo: below we need to redesign the process to match three types of file.
+            // 1. empty file: write into the data_map the string EMPTY. On decrypt, we detect this and just create an empty file.
+            // 2. A small file: The whole content of those can be in the data_map. Just in SE lib we write the < MIN_SIZE to the data map.
```

Comment:
the `MIN_ENCRYPTABLE_BYTES` is defined by the encryption crate, which is 3 bytes.
then within this branch you can just do:
```rust
if file_size < MIN_ENCRYPTABLE_BYTES as u64 {
    let content: Bytes = File::read(file_path);
    let chunk = Chunk::new(content);
    (chunk.name(), chunk, vec![])
}
```

Note this is to generate a Chunk with data_content, instead of data_map
Hence on the `parse` side, need to double check the code whether the case is covered as well.

If you track the commit history way back (Dec 2023 shall be enough), there shall be code handling `write small file to data_map` still presented